src/RuleSet/Sets/PhpCsFixerRiskySet.php

```diff
@@ -51,6 +51,7 @@ public function getRules(): array
             'no_unset_on_property' => true,
             'php_unit_data_provider_name' => true,
             'php_unit_data_provider_return_type' => true,
+            'php_unit_data_provider_static' => true,
```

Comment:
```suggestion
            'php_unit_data_provider_static' => [force' => true],
```

- if we want to have it in, it should be forceful
- also to avoid config change between `php_unit_data_provider_static` enabled via Fixer ruleset or PHPUnit migration ruleset.

Reply:
Good point 👍